async/logic/Async4BackendRest_CallbackAPI_TopicCheckLogic.java

```diff
@@ -0,0 +1,343 @@
+				System.out.println("Async4BackendRest_CallbackAPI_Topic redo");
+				continue;
+			}
+			
+			System.out.println("Async4BackendRest_CallbackAPI_Topic messages picked");
+			System.out.println("Test1");
+
+			typeresponse = ExecuteCallBackAPI(callbackUri, destinationStatusCode, destinationResponseBody, transactionid, asyncid, executiontime, destinationresult);
+			System.out.println("Test2");
+			if (((Response) typeresponse).getStatus() != Response.Status.CREATED.getStatusCode()) {
+				new AsyncResult<Response>(typeresponse);
+
+			}else{
+				
+			}
+			
+			
+			// 1秒待つ
+			try {
+				TimeUnit.SECONDS.sleep(1);
+				System.out.println("Test");
+				//return new AsyncResult<Response>(typeresponse);
+			} catch (InterruptedException e) {
+				e.printStackTrace();
+			}
+			
+		}
+		
+	}
+	
+	private Response PickupMessageForResponseQueue(StringBuilder destinationStatusCode, StringBuilder destinationResponseBody, 
+	StringBuilder callbackUri, StringBuilder destinationresult, StringBuilder transactionid, StringBuilder asyncid, 
+	StringBuilder executiontime) {
+	    Responses response      = new Responses();
+	    String receiveMessage = new String();
+	    
+	    destinationStatusCode.delete(0, destinationStatusCode.length());
+    	destinationResponseBody.delete(0, destinationResponseBody.length());
+    	callbackUri.delete(0, callbackUri.length());
+		destinationresult.delete(0, destinationresult.length());
+		transactionid.delete(0, transactionid.length());
+		asyncid.delete(0, asyncid.length());
+		executiontime.delete(0, executiontime.length());
+    	
+	    // 接続時の設定値を Properties インスタンスとして構築する
+	    Properties properties = new Properties();
+	    // 接続先 Kafka ノード
+	    properties.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, "asynckafka1:9092");
+	    // Consumer を識別するための group id
+	    properties.put(ConsumerConfig.GROUP_ID_CONFIG, "java-consumer-group");
+	    // 読み取り位置である offset が未指定だった場合に、Kafka 上に最も早く置かれたメッセージを読み取る
+	    properties.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+	    // 読み取り後に、読み取り位置である offset を自動で更新する
+	    properties.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "false");
+	    // 最大1つのメッセージを取得する。
+	    properties.put(ConsumerConfig.MAX_POLL_RECORDS_CONFIG, 1);
+	 
+	    // Consumer を構築する
+	    KafkaConsumer<String, String> consumer =
+	        new KafkaConsumer<String, String>(properties, new StringDeserializer(), new StringDeserializer());
+	 
+	    // Consumer をトピックに割り当てる
+	    consumer.subscribe(Arrays.asList("Async4BackendRest_CallbackAPI_Topic"));
+	 
+	    try {
+	      while (true) {
+	        // メッセージをとりだす
+	        ConsumerRecords<String, String> records = consumer.poll(Duration.ofSeconds(60l));
+	        Date date = new Date();
+    		SimpleDateFormat dateformat = new SimpleDateFormat("yyyy/MM/dd HH:mm:ss");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you do not call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.